.trunk/trunk.yaml

```diff
@@ -0,0 +1,62 @@
+# This file controls the behavior of Trunk: https://docs.trunk.io/cli
+# To learn more about the format of this file, see https://docs.trunk.io/reference/trunk-yaml
+version: 0.1
+cli:
+    version: 1.22.2
+# Trunk provides extensibility via plugins. (https://docs.trunk.io/plugins)
+plugins:
+    sources:
+        - id: trunk
+          uri: https://github.com/trunk-io/plugins
+          ref: v1.6.0
+
+# Many linters and tools depend on runtimes - configure them here. (https://docs.trunk.io/runtimes)
+runtimes:
+    enabled:
+        - go@1.21.0
+        - node@18.12.1
+        - python@3.10.8
```

Comment:
[Question] Does this mean that in order to develop on ESLint locally, we'll need Go and Python runtimes?

(asking here explicitly rather than trying myself to get the answer visible - which I think is valuable either way)

Reply:
Trunk will hermetically manage any runtime it needs for the linters it runs. This is completely invisible to the user. They will not need to install anything on their system.

Reply:
Do you have public docs on how this works? I'm reading through https://docs.trunk.io/check/reference/under-the-hood & https://docs.trunk.io/check/faq but not finding it.

Reply:
We have https://docs.trunk.io/check/reference/trunk-yaml#runtimes and https://docs.trunk.io/check/advanced-setup/tools/configuration#package-based-tools.

Trunk downloads a tarball specific to your platform for each runtime and uses the usual tools specific to them to install packages for a specified linter (For example, `pip`, for Python).